src/Enterspeed.Cli/Services/IngestService/IngestService.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.Text;
+using System.Text.Json;
+using Microsoft.Extensions.Logging;
+
+namespace Enterspeed.Cli.Services.IngestService;
+
+public class IngestService : IIngestService
+{
+    private readonly HttpClient _httpClient;
+
+    private readonly ILogger<IngestService> _logger;
+    public IngestService(ILogger<IngestService> logger, ISettingsService settingsService)
+    {
+        _logger = logger;
+
+        _httpClient = new HttpClient
```

Comment:
The lifecycle of the CLI is different from a web application, so maybe this is not an issues.

But new instances of the http client should not be created everytime. The IHttpClientFactory could be used to keep the state of http clients: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-7.0

Reply:
I don't think it's an issue, as the IngestService will only create a single HttpClient before exiting.